web/src/views/DandisetLandingView/DandisetLandingView.vue

```diff
@@ -20,15 +20,7 @@
     </template>
     <template v-else>
       <v-toolbar class="grey darken-2 white--text">
-        <v-btn
-          icon
-          @click="navigateBack"
-        >
-          <v-icon color="white">
-            mdi-arrow-left
-          </v-icon>
-        </v-btn>
-        <v-toolbar-title>
+        <v-toolbar-title class="d-none d-md-block mx-8">
```

Comment:
If we keep the grey bar, I think the margin here should be removed, as it looks awkward.
```suggestion
        <v-toolbar-title class="d-none d-md-block">
```

Reply:
I agree that this looks better. The `mx-8` class is on the public dandisets page as well, and I was copying that to maintain consistent style.

I am going to change those on a separate PR so that Jared can advise us on the design impacts.

Reply:
FYI, I included this fix with #703.